docs/concepts/abstractions/overview.md

```diff
@@ -0,0 +1,58 @@
+---
+---
+
+{% capture overview %}
+This page explains how Kubernetes objects are represented in the Kubernetes API, and how you can express them in `.yaml` format.
+{% endcapture %}
+
+{% capture body %}
+### Understanding Kubernetes Objects
+
+*Kubernetes Objects* are persistent entities in the Kubernetes system. Kubenetes uses these entities to represent the state of your cluster. Specifically, they can describe:
+
+* What containerized applications are running (and on which nodes)
+* The resources available to those applications
+* The policies around how those applications behave, such as restart policies, upgrades, and fault-tolerance
+
+When you create a Kubernetes object, you create a "record of intent"--once you create the object, the Kubernetes system will constantly work to ensure that the entity exists. By creating an object, you're effectively telling the Kubernetes system what you want your cluster to be doing; this is your cluster's **desired state**.
```

Comment:
entity in this case is the same as "object". I think I cribbed the wording from somewhere else. I can just changed "entity" to "object" here.